histomicstk/cli/NucleiDetection/NucleiDetection.xml

```diff
@@ -159,6 +159,13 @@
       <description>Minimum area that each nucleus should have</description>
       <longflag>min_nucleus_area</longflag>
       <default>80</default>
+    </double>
```

Comment:
If this is in pixels, it really should be an integer.  I'd move it to immediately after the `analysis_tile_size` parameter.  (And that parameter, too, really should be an integer).

Reply:
Yes, I agree. I checked the function which takes this as an argument. It was also expecting an integer. I've updated the code with these changes.